CIP-0035/README.md

```diff
@@ -156,26 +181,22 @@ So, for example, a single CIP could propose multiple new builtin functions, or a
 
 Proposals for additions to the set of Plutus Core builtins SHOULD be proposed in a CIP and SHOULD adhere to the following additional process.
 
-In order to move to Draft status, it MUST include:
+In order to move to Proposed status, it MUST include:
 - In the Specification:
     - Names and types/kinds for the new functions or types.
     - A source for the implementation (e.g. a library which can be linked against); or a generic description of the functionality which is implementable in any programming language.
-    - In the Rationale:
+    - For new types: a precise description of the measure used for the size of a value of that type.
+    - For new builtin functions: a costing function for the builtin function.
+- In the Rationale:
     - An argument for the utility of the new builtins.
+    - If an external implementation is provided: an argument that it is trustworthy.
+    - Discussion of how any measures and costing functions were determined.
```

Comment:
I think I'm deliberately leaving that a little vague because I'm not sure. I think I want them to _talk_ about it but I think we might want more or less depending on the proposal :thinking:

Reply:
Also: I think those benchmarks would be part of the implementation, which is separate from the CIP.